content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -229,7 +232,18 @@ class Main
 
 <PySection>
 
-In order to sort a list, simply use the `sorted(list)` function.
+In order to sort a list, use the `arr.sort()` function. To get a copy of the list in a sorted order, use `sorted(arr)`
+	
+```python
+arr = [5, 1, 3, 2, 4]
+arr.sort()
+print(arr)  # [1, 2, 3, 4, 5]
+	
+arr = [5, 1, 3, 2, 4]
+sorted_arr = sorted(arr) 
+print(arr)  # [5, 1, 3, 2, 4]
+print(sorted_arr)  # [1, 2, 3, 4, 5]
+```
 
 - [Sorting Basics](https://docs.python.org/3/howto/sorting.html)
```

Comment:
Not sure if this line is needed since none of the other languages have something like this

Reply:
It is useful to mention. There are probably times when you would need a copy of a sorted array.